src/App.js

```diff
@@ -14,11 +17,64 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+export default class App extends React.Component {
+  state = {
+    selectedGoods: [],
+  }
 
-export default App;
+  electNextGoods = (e) => {
+    const { selectedGoods } = this.state;
+
+    if (e.ctrlKey) {
+      this.setState({ selectedGoods: [...selectedGoods, e.target.innerHTML] });
```

Comment:
Тут надо проверять нет ли уже этого элемента в массиве, либо удалять его, либо добавлять в зависимости от этого
```suggestion
      this.setState((prevState) => ({ selectedGoods: [...prevState.selectedGoods, e.target.innerHTML] }));
```

Reply:
чому варіант який я зробив не підходить ?